src/libstore/unix/include/nix/build/local-derivation-goal.hh

```diff
@@ -1,7 +1,7 @@
 #pragma once
 ///@file
 
-#include "derivation-goal.hh"
+#include "nix/build/derivation-goal.hh"
```

Comment:
Are those renames file a leftover or intended?

Reply:
I mean that the local-derivation-goal.hh header is no longer in the same place as the c++ file.

Reply:
These renames are intended.

 - `nix/` is definitely intended
 - `build/` is an artifact of how all of that used to be in one giant `build.cc`. It is unclear to me how much we exposed it before. I say, either expose it or get rid of it.